applications/derive_mirror_rnda.py

```diff
@@ -174,23 +176,26 @@ def _parse(label):
         "--mirror_list",
         help=(
             "Mirror list file to replace the default one. It should be used if"
-            " measured mirror focal lengths need to be accounted"
+            " measured mirror focal lengths need to be accounted. It contains the following"
```

Comment:
I think we should not describe the columns and data fields of the input files and keep the help messages short (if possible to a single line).

Reply:
I agree. the problem here is that there is no doctring for the function and if we just remove this explanation we lose it for good. I will create a docstring for the function and include the detailed information there and keep it small in the help message

Reply:
Ok, so I reduced the size and did not add a docstring (I actually think it would not be necessary)